### PR TITLE
fix(localization): Word and grammaring adjustment

### PR DESCRIPTION
**Pull request by satoufuyuki**

### CODE REVIEW

**Comment by satoufuyuki** [comment]
@mzrtamp if you want to improve word, grammar  and markdown, please commit to this PR

File: lang/id.json [hunk 1]
```diff
@@ -23,29 +23,29 @@
...
-				"slashDescription": "Nama command untuk melihat informasi tentang command tertentu",
-				"usage": "{prefix}help [command]",
-				"authorString": "{username} - Daftar Command",
-				"footerString": "{prefix}help <command> untuk mendapatkan informasi tentang command tertentu",
-				"noCommand": "Tidak dapat menemukan command dengan nama tersebut",
-				"commandSelectionString": "Silahkan pilih command",
-				"noCommanSuggest": "Tidak dapat menemukan command dengan nama tersebut. Apakah ini yang kamu maksud?",
-				"commandDetailTitle": "{username} - Informasi tentang command {command}",
+				"description": "Menunjukkan daftar perintah atau informasi tentang perintah tertentu",
+				"slashDescription": "Nama commanperintahd untuk melihat informasi tentang perintah tertentu",
```
Comment (Mednoob) [inline lang/id.json]: Line 27 typo

**Review by Mednoob — CHANGES_REQUESTED** [review]

**Comment by mzrtamp** [comment]
> @mzrtamp if you want to improve word, grammar and markdown, please commit to this PR

Ok

**Review by mzrtamp — COMMENTED** [review]
Let me propose 'em

**Review by mzrtamp — APPROVED** [review]
Chotto matte-

**Review by Mednoob — APPROVED** [review]

**Review by mzrtamp — APPROVED** [review]
That's all, maybe?

**Pull request merged**